canonical-web-and-design/maas-ui
# Add and edit users over the API

Done:
- Add sagas/actions/reducers for creating/updating users.
- Connect the user create/update forms to the API. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1373.

QA:
- Navigate to the users list.
- Click on the Add user button.
- Fill out the form and submit.
- You should be redirected to the user list and the new user should be in the list.
- Repeat with editing a user.
Note: a message should appear when you are redirected: https://github.com/canonical-web-and-design/maas-ui/issues/108.
